client/my-sites/pages/page-list.jsx

```diff
@@ -31,54 +27,47 @@ import {
 	getSelectedSiteId,
 } from 'state/ui/selectors';
 
-var PageList = React.createClass( {
-
-	mixins: [ PureRenderMixin ],
-
-	propTypes: {
-		context: React.PropTypes.object,
-		search: React.PropTypes.string,
-		hasSites: React.PropTypes.bool.isRequired,
-		site: React.PropTypes.object,
-		siteId: React.PropTypes.any
-	},
-
-	render: function() {
-		return (
-			<PostListFetcher
-				type="page"
-				number={ 100 }
-				siteId={ this.props.siteId }
-				status={ mapStatus( this.props.status ) }
-				search={ this.props.search }>
-				<Pages
-					{ ...omit( this.props, 'children' ) }
-				/>
-			</PostListFetcher>
-		);
-	}
-} );
-
-var Pages = React.createClass( {
+const PageList = ( props ) => (
+	<PostListFetcher
+		type="page"
+		number={ 100 }
+		siteId={ props.siteId }
+		status={ mapStatus( props.status ) }
+		search={ props.search }>
+		<Pages
+			{ ...omit( props, 'children' ) }
+		/>
+	</PostListFetcher>
+);
+
+PageList.propTypes = {
+	context: PropTypes.object,
+	search: PropTypes.string,
+	hasSites: PropTypes.bool.isRequired,
+	site: PropTypes.object,
+	siteId: PropTypes.any
+};
+
+const Pages = localize( React.createClass( {
```

Comment:
That `infiniteScroll` mixin - we're currently using it at just 2 locations, perhaps it would be a wise solution to refactor it as a HoC and get rid of the mixin. But until then, ¯\\_(ツ)_/¯

Perhaps that task can be included in the reduxification effort @ockham?